apps/console/src/features/identity-providers/components/forms/authenticators/saml-authenticator-settings-form.tsx

```diff
@@ -0,0 +1,814 @@
+import { TestableComponentInterface } from "@wso2is/core/models";
+import { SCheckboxAdapter, SSelectFieldAdapter, STextFieldAdapter } from "@wso2is/form";
```

Comment:
So we need to have the `S` prefix here. IMO, users do not need to know if we are using `Semantic` in the adapters.

Reply:
The `S` prefix is there to denote `Simplified`, not `Semantic`. Also, since there are similar exports in the form module we need this prefix to avoid collision with other named exports. With that, the intention of this approach is to grant full control to the form consumer.